queries.sql

```diff
@@ -7,3 +7,38 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+
+
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
```

Comment:
- [ ] According to the project requirement, you have to add a column **species** of type **string** to your animals table in your schema.sql file. Please fix it and make sure that your queries are implemented and work as expected and please add a correct screenshot with updating in your PR :smile:

Reply:
Thanks @codecaiine for detecting this issue , I have made the stated corrections, hope my project should be approved by the next code reviewer.

Reply:
@sonyco-4u I'm trying to reach out to you on slack